sources/host-ctr/cmd/host-ctr/main.go

```diff
@@ -321,6 +321,11 @@ func withSuperpowered(superpowered bool) oci.SpecOpts {
 				Options:     []string{"rbind", "ro"},
 				Destination: "/usr/src/kernels",
 				Source:      "/usr/src/kernels",
+			},
+			{
+				Options:     []string{"rbind"},
```

Comment:
It doesn't seem like we'd need this read/write.  What do you think?  Want to try with `ro`?

Reply:
Since superpowered containers are meant as a break-glass mechanism, I'm fine with allowing writes to debugfs.

The other read-only mounts are all for paths that are also read-only in the host, because they reside on the dm-verity backed root filesystem.